cmd/scollector/collectors/interval.go

```diff
@@ -48,10 +49,15 @@ func (c *IntervalCollector) Run(dpchan chan<- *opentsdb.DataPoint) {
 		}
 		next := time.After(interval)
 		if c.Enabled() {
+			timeStart := time.Now()
 			md, err := c.F()
+			timeFinish := time.Now()
+			result := "success"
```

Comment:
Result needs to be a timeseries value we always report instead of a tag. If it's a tag, then it's too easy for unknowns to happen. So make another metric called `scollector.collector.error` and have it be 1 or 0 for error (1) or not (0).